include/klee/Core/ExecutionState.h

```diff
@@ -10,13 +10,12 @@
 #ifndef KLEE_EXECUTIONSTATE_H
 #define KLEE_EXECUTIONSTATE_H
 
-#include "klee/Constraints.h"
-#include "klee/Expr.h"
-#include "klee/Internal/ADT/TreeStream.h"
-
-// FIXME: We do not want to be exposing these? :(
 #include "../../lib/Core/AddressSpace.h"
```

Comment:
Don't drop the FIXME if you don't actually fix the problem. So either put the FIXME back on move `AddressSpace.h` into `include/klee/Core` so it can be included as `#include "klee/Core/AddressSpace.h"`